presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/converter/GlueStatConverter.java

```diff
@@ -0,0 +1,315 @@
+
+            case DOUBLE:
+                DoubleColumnStatisticsData catalogDoubleData = catalogColumnStatisticsData.getDoubleColumnStatisticsData();
+                OptionalDouble minOfDouble = OptionalDouble.of(catalogDoubleData.getMinimumValue());
+                OptionalDouble maxOfDouble = OptionalDouble.of(catalogDoubleData.getMaximumValue());
+                OptionalLong nullsCountOfDouble = fromMetastoreNullsCount(catalogDoubleData.getNumberOfNulls());
+                OptionalLong distinctValuesCountOfDouble = OptionalLong.of(catalogDoubleData.getNumberOfDistinctValues());
+                return createDoubleColumnStatistics(minOfDouble, maxOfDouble, nullsCountOfDouble, fromMetastoreDistinctValuesCount(distinctValuesCountOfDouble, nullsCountOfDouble, rowCount));
+
+            case LONG:
+                LongColumnStatisticsData catalogLongData = catalogColumnStatisticsData.getLongColumnStatisticsData();
+                OptionalLong minOfLong = OptionalLong.of(catalogLongData.getMinimumValue());
+                OptionalLong maxOfLong = OptionalLong.of(catalogLongData.getMaximumValue());
+                OptionalLong nullsCountOfLong = fromMetastoreNullsCount(catalogLongData.getNumberOfNulls());
+                OptionalLong distinctValuesCountOfLong = OptionalLong.of(catalogLongData.getNumberOfDistinctValues());
+                return createIntegerColumnStatistics(minOfLong, maxOfLong, nullsCountOfLong, fromMetastoreDistinctValuesCount(distinctValuesCountOfLong, nullsCountOfLong, rowCount));
+
+            case STRING:
+                StringColumnStatisticsData catalogStringData = catalogColumnStatisticsData.getStringColumnStatisticsData();
+                OptionalLong maxColumnLengthOfString = OptionalLong.of(catalogStringData.getMaximumLength());
+                OptionalDouble averageColumnLengthOfString = OptionalDouble.of(catalogStringData.getAverageLength());
+                OptionalLong nullsCountOfString = fromMetastoreNullsCount(catalogStringData.getNumberOfNulls());
+                OptionalLong distinctValuesCountOfString = OptionalLong.of(catalogStringData.getNumberOfDistinctValues());
+
+                return createStringColumnStatistics(
+                        maxColumnLengthOfString,
+                        getTotalSizeInBytes(averageColumnLengthOfString, rowCount, nullsCountOfString),
+                        nullsCountOfString,
+                        fromMetastoreDistinctValuesCount(distinctValuesCountOfString, nullsCountOfString, rowCount));
+        }
+
+        throw new PrestoException(HIVE_INVALID_METADATA, "Invalid column statistics data: " + catalogColumnStatisticsData);
+    }
+
+    private static ColumnStatisticsData getGlueColumnStatisticsData(HiveColumnStatistics statistics, HiveType columnType, OptionalLong rowCount)
```

Comment:
```suggestion
    private static ColumnStatisticsData toGlueColumnStatisticsData(HiveColumnStatistics statistics, HiveType columnType, OptionalLong rowCount)
```

nit: Rename the method to follow the `fromXXX` and `toXXX` pattern.